docs/src/pages/guides/DB2OnCloudSetup.md

```diff
@@ -2,146 +2,173 @@
-        <jdbcDriver libraryRef="fhirSharedLib"/>
-        <properties.db2.jcc
-            serverName="dashdb-txn-flex-************.services.dal.bluemix.net"
-            portNumber="50001"
-            apiKey="<API-KEY>"
-            securityMechanism="15"
-            pluginName="IBMIAMauth"
-            databaseName="BLUDB"
-            currentSchema="JBATCH"
-            driverType="4" sslConnection="true" sslTrustStoreLocation="resources/security/dbTruststore.jks" sslTrustStorePassword="<TRUSTSTORE-PASSWORD>"/>
-    </dataSource>
+### **Testing the Connection**
 
-    <batchPersistence jobStoreRef="BatchDatabaseStore" />
-    <databaseStore id="BatchDatabaseStore" dataSourceRef="fhirbatchDS" schema="JBATCH" tablePrefix="" />
-```
+The [Db2 driver](https://repo1.maven.org/maven2/com/ibm/db2/jcc/11.5.0.0/jcc-11.5.0.0.jar) is able to execute a connectivity test to check the configuration of the combo of API-key/Service-Id/Db2-User-Id.
+
+1. Copy the command to your code editor
 
-Note, the Java Batch is configured in batchDs.xml  and included from the default server.xml that gets installed to the `{wlp}/usr/server/fhir-server`.
+    ``` bash
+    java -cp /path/to/db2jcc4.jar com.ibm.db2.jcc.DB2Jcc  -url "jdbc:db2://<DB2-HOSTNAME>:50001/BLUDB:apiKey=<API-KEY>;securityMechanism=15;sslConnection=true;sslTrustStoreLocation=/path/to/truststore.jks;sslTrustStorePassword=<TRUSTSTORE-PASSWORD>;"
+    ```
 
-### Configuring FHIR Datasource
+    - Note: Don't forget the trailing `;` in the URL. Some of the documented examples don't include it, but it is required in order for the connection to work, although this may be fixed in a future driver release. This only affects this test URL, not the actual FHIR server configuration.
 
-The FHIR server uses a proxy datasource mechanism, allowing new datasources to be added at runtime without requiring a (Liberty Profile) server restart. To configure a FHIR tenant datasource using an API-KEY, use the following template:
+1. Replace the following values with your service details:
+    - `/path/to/db2jcc4.jar` : replace with the path to your driver jar. 
+    - `<DB-HOSTNAME>`: the hostname of your Db2 service from the Service Credentials page
+    - `<API-KEY>`: the API key value created in the previous section
+    - `<TRUSTSTORE-PASSWORD>`: the password for your truststore
+   
+    - Note: When using an API Key, no username needs to be provided. This is because the API Key maps to a ServiceId, and that ServiceId is mapped to the Db2 user.
+
+1. Run in your favorite terminal, and you should see no errors in the output. You should see output like: 
+
+```
+[jcc][10516][13709]Test Connection Successful.
+```
+
+### **Configuring IBM FHIR Server Datasource**
+
+The IBM FHIR server uses a proxy datasource mechanism, allowing new datasources to be added at runtime without requiring a (Liberty Profile) server restart. To configure a FHIR tenant datasource using an API-KEY, use the following template:
```

Comment:
worth mentioning which file to put this in and where to find it?  or maybe just link to that section of the user's guide?

Reply:
I've now added.